Packs/Lastline/Integrations/Lastline_v2/Lastline_v2.py

```diff
@@ -4,6 +4,7 @@
 import demistomock as demisto
 from CommonServerPython import *
 from CommonServerUserPython import *
+from datetime import datetime
```

Comment:
I don't think you need it... you get it from common server python

Reply:
removed